fuzzers/aflcc/fuzzer.py

```diff
@@ -0,0 +1,344 @@
+    """Add additional linking flags for certain benchmarks"""
+    if is_benchmark('libjpeg'):
+        ldflags_arr += libjpeg_turbo_asm_object_files()
+    elif is_benchmark('php'):
+        ldflags_arr += ['-lresolv']
+    elif is_benchmark('curl'):
+        ldflags_arr += ['-ldl', '-lpsl', '/src/openssl/libcrypto.a', '/src/openssl/libssl.a']
+    elif is_benchmark('openssl'):
+        ldflags_arr += ['/src/openssl/libcrypto.a', '/src/openssl/libssl.a']
+    elif is_benchmark('systemd'):
+        shutil.copy(os.path.join(os.environ['OUT'], 'src/shared/libsystemd-shared-245.so'), '/usr/lib/libsystemd-shared-245.so')
+        ldflags_arr += ['-lsystemd-shared-245']
+
+def prepare_fuzz_environment(input_corpus):
+    """Prepare run for some benchmarks"""
+    afl_fuzzer.prepare_fuzz_environment(input_corpus)
+
+    # OUT env variable does not exists, it seems
+    if os.path.isfile('/out/src/shared/libsystemd-shared-245.so'):
+        shutil.copy('/out/src/shared/libsystemd-shared-245.so', '/usr/lib/libsystemd-shared-245.so')
+
+def prepare_build_environment():
+    """Set environment variables used to build benchmark."""
+    utils.set_no_sanitizer_compilation_flags()
+
+    # Update compiler flags for clang-3.8.
+    cflags = ['-fPIC']
+    cppflags = cflags + ['-I/usr/local/include/c++/v1/', 
+                         '-stdlib=libc++', '-std=c++11']
+    utils.append_flags('CFLAGS', cflags)
+    utils.append_flags('CXXFLAGS', cppflags)
+
+    # Add flags for various benchmarks
+    add_compilation_cflags()
+
+    # Setup aflcc compiler
+    os.environ['LLVM_CONFIG'] = 'llvm-config-3.8'
+    os.environ['CC'] = '/afl/aflc-gclang'
+    os.environ['CXX'] = '/afl/aflc-gclang++'
+    #  -L/ -lAflccMock -lpthread
+    os.environ['FUZZER_LIB'] = '/libAFL.a'
+
+    # Fix FUZZER_LIB for various benchmarks
+    fix_fuzzer_lib()
+
+def get_fuzz_targets():
+    """Get the fuzz target name"""
+    # For non oss-projects, FUZZ_TARGET contain the target binary
+    fuzz_target = os.getenv('FUZZ_TARGET', None)
+    if fuzz_target is not None:
+        return [fuzz_target]
+
+    print('FUZZ_TARGET is not defined')
+
+    # for these benchmarks, only return one file
+    targets = {'curl': 'curl_fuzzer_http', 
+               'openssl': 'x509',
+               'systemd': 'fuzz-link-parser',
+               'php': 'php-fuzz-parser'}
+
+    for target,fuzzname in targets.items():
+        if is_benchmark(target):
+            return [os.path.join(os.environ['OUT'], fuzzname)]
+
+    # For the reamining oss-projects, use some heuristics.
+    # We look for binaries in the OUT directory and take it as our targets.
+    # Note that we may return multiple binaries: this is necessary because
+    # sometimes multiple binaries are generated and we don't know which will
+    # be used for fuzzing (e.g., zlib benchmark)
+    # TODO(laurentsimon): hardcode targets for oss-fuzz projects in the 'targets' dictionary above
+    out_dir = os.getenv('OUT', None)
```

Comment:
I think instead of using getenv and checking `if None` and excepting if is, you could just use os.environ. It will do the same thing.